wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/TaskDefinitionElementToWomTaskDefinition.scala

```diff
@@ -57,8 +57,20 @@ object TaskDefinitionElementToWomTaskDefinition {
         }.map(_.toSeq)
       }
 
-      (validRuntimeAttributes, validCommand) mapN { (runtime, command) =>
-        CallableTaskDefinition(a.taskDefinitionElement.name, Function.const(command.validNel), runtime, Map.empty, Map.empty, taskGraph.outputs, taskGraph.inputs, Set.empty, Map.empty)
+      val metaSection : ErrorOr[Map[String, String]] = a.taskDefinitionElement.metaSection match {
+        case None => Map.empty.validNel
+        case Some(MetaSectionElement(meta)) =>
+          val m : Map[String, String] = meta.map{
+            case (key, MetaValueElement.MetaValueElementString(value)) =>
+              key -> value
+            case (key, other) =>
+              throw new Exception(s"non string meta values are not handled currently <${key} -> ${other}>")
```

Comment:
This is a great find! The WDL 1.0 spec says we should support compound types and `wom.callable.MetaValueElement` has these types implemented, yet we can't wire them through because `wom.callable.CallableTaskDefinition#meta` is still of type `Map[String, String]`

I filed issue https://github.com/broadinstitute/cromwell/issues/4746 - I think it would be helpful to include a reference to it in this error message so people know what's up.